Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -359,21 +365,21 @@ private sealed class ExecuteRecipeShortcut { }
 
                     if (recipe == null)
                     {
-                        throw new RecipeNotFoundException($"{recipeName} not found!");
+                        throw new RecipeNotFoundException($"Recipe with the name {recipeName} not found.");
```

Comment:
Why I didn't do this is because this is a dot-separated technical name, so (as opposed to display names that can contain spaces) it's apparent what's the name (which is actually more like an ID). But for consistency, we can have it with quotes.